### ISSUE TITLE
[SUGGESTION] Cartridge model

### ISSUE THREAD

**Issue by JoeJoeTV** [NPException42/NPArcade#12]
I made a NES Cartridge model and i think it fits this mod.
(In the attached zip is a vanilla resource pack(mc1.12.2)
 with the carrot on a stick replaced with this model and the raw model and texture)
[gitnescart.zip](https://github.com/NPException/NPArcade/files/2137837/gitnescart.zip)
P.S. The Cartridge Logo is a different texture than the main one

**Comment by NPException** [comment]
I've never used a resource pack before, can you tell me how I can view the model?

**Comment by NPException** [comment]
Also note, that cartridges right now are already based on the NES cartridge design, but just very low resolution / detail.
The current cartridge model is just in bright grayscale colors and does not have a logo on it. The color of the cartridge and the logo on it in game is defined by the games.

**Comment by JoeJoeTV** [comment]
Yes i made the model for that pupose, so you can change it easily.
And to view the model, just put the resource pack in the resource pack folder and select it ingame.
I can also send the model in different file formats.
You can also use a Minecraft Model Editor like blockbench to view the model, but then you need to reload the texture, so it is displayed correctly.
